units/src/amount/signed.rs

```diff
@@ -439,7 +454,8 @@ impl SignedAmount {
         if self.is_negative() {
             Err(OutOfRangeError::negative())
         } else {
-            Ok(Amount::from_sat(self.to_sat() as u64)) // Cast ok, checked not negative above.
+            // Unchecked because a positive signed amount is always valid.
```

Comment:
This seems to say the exact thing as the comment on the next line.  Don't need both of these comments imo.